ha/core/controllers/controller_annotation.py

```diff
@@ -16,14 +16,17 @@
 # cortx-questions@seagate.com.
 
 import json
+import traceback
 from ha import const
+from cortx.utils.log import Log
 
 def controller_error_handler(func):
     def inner_function(*args, **kwargs) -> str:
         try:
             result: dict = func(*args, **kwargs)
             return json.dumps(result)
         except Exception as e:
+            Log.error(f"ClusterManagerException. {func.__name__} failed. {traceback.format_exc()}, {e}")
```

Comment:
Is this used only in context of Cluster Manager? Should not have such assumption as this is in different directory.

Reply:
`Is this used only in context of Cluster Manager?`
It is cluster manager action i.e. controller
If any controller throw exception this covert them to 
```
 {"status": const.STATUSES.FAILED.value,
                "msg": f"ClusterManagerException. {func.__name__} failed. Error: {e}"}
```

Reply:
`Is this used only in context of Cluster Manager?`
It is useful if controller action throw any exception
It convert them to 
```
{"status": const.STATUSES.FAILED.value,
                "msg": f"ClusterManagerException. {func.__name__} failed. Error: {e}"}
```